pydicom/uid.py

```diff
@@ -172,15 +170,11 @@ def generate_uid(prefix=pydicom_root_UID, truncate=False):
     max_uid_len = 64
 
     if prefix is None:
-        dicom_uid = '2.25.{0}'.format(uuid.uuid1().int)
-    else:
-        uid_info = [uuid.getnode(),
-                    fabs(os.getpid()),
-                    datetime.datetime.today().second,
-                    datetime.datetime.today().microsecond]  # nopep8
-
-        suffix = ''.join([str(int(x)) for x in uid_info])
-        dicom_uid = ''.join([prefix, suffix])
+        prefix = '2.25.'
+
+    suffix = uuid.uuid1().int
+
+    dicom_uid = ''.join([prefix, str(suffix).lstrip("0")])
```

Comment:
Why are you left stripping 0s? How can the str representation of an int have leading 0s (other than for the number 0, which is a valid UID value).